Tasks/HelmInstallerV1/src/utils.ts

```diff
@@ -0,0 +1,102 @@
+"use strict";
+
+import tl = require('vsts-task-lib/task');
+import path = require('path');
+import fs = require('fs');
+import * as toolLib from 'vsts-task-tool-lib/tool';
+import * as utils from './utils';
+import * as os from "os";
+import * as util from "util";
+import { WebRequest, sendRequest } from 'utility-common/restutilities';
+
+const uuidV4 = require('uuid/v4');
```

Comment:
Should not we be using the common code? Why clone the code?

Reply:
used common code